### ISSUE TITLE
Error after update to latest version

### ISSUE THREAD

**Issue by evilebottnawi** [sarbbottam/eslint-find-rules#280]
```shell
/home/travis/build/itgalaxy/eslint-plugin-itgalaxy/node_modules/define-property/index.js:14
     throw new TypeError('expected an object or function.');
     ^
 
 TypeError: expected an object or function.
     at defineProperty (/home/travis/build/itgalaxy/eslint-plugin-itgalaxy/node_modules/define-property/index.js:14:11)
     at Object.<anonymous> (/home/travis/build/itgalaxy/eslint-plugin-itgalaxy/node_modules/window-size/index.js:125:1)
     at Module._compile (module.js:571:32)
     at Object.Module._extensions..js (module.js:580:10)
     at Module.load (module.js:488:32)
     at tryModuleLoad (module.js:447:12)
     at Function.Module._load (module.js:439:3)
     at Module.require (module.js:498:17)
     at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/travis/build/itgalaxy/eslint-plugin-itgalaxy/node_modules/eslint-find-rules/dist/lib/cli-util.js:5:12)
```

**Comment by ta2edchimp** [comment]
I just had a quick look into this, but the issue seems to be with `window-size` [see here](https://github.com/jonschlinkert/window-size/issues/11) and related to the tests running in Travis.  
In our tests, that module's response is mocked, and therefore is not failing on Travis.

I have not yet an idea how to handle this, except, maybe downgrading the dependency again, until the issue is solved on their end.

**Comment by ljharb** [comment]
Downgrading and pinning seems correct.

**Comment by ta2edchimp** [comment]
Fixed with [`eslint-find-rules@3.1.0`](https://www.npmjs.com/package/eslint-find-rules).
